release_images.yml

```diff
@@ -1,3 +1,17 @@
 ---
 release_images:
-# Check .release_images_template.yml for reference
\ No newline at end of file
+# Check .release_images_template.yml for reference
+  1:
+    framework: "pytorch"
+    version: "1.10.0"
+    arch_type: "x86"
```

Comment:
you'll need to use customer type - one defintion for "e3" and one for "sagemaker"